packages/presets/src/fragments/copilot-panel/utils/markdown-utils.ts

```diff
@@ -6,8 +6,12 @@ import { Job, type Slice } from '@blocksuite/store';
 
 export async function getMarkdownFromSlice(host: EditorHost, slice: Slice) {
   const job = new Job({ collection: host.std.doc.collection });
+  const snapshot = await job.sliceToSnapshot(slice);
```

Comment:
Why do you need this change?

Reply:
These changes are introduced when resolving conflicts. Let me see if I can remove them.